docs/modules/clusters/pages/ucn-static-config.adoc

```diff
@@ -0,0 +1,28 @@
+= Static Configuration
+:description: You can configure {ucn} using static configuration in an XML or YAML file, or programatically. The configuration is applied on start-up. 
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+To statically configure {ucn} for your user code, you must do the following:
+
+* If not already done, xref:clusters:ucn-enable.adoc[]
+* Define the name of the {ucn}
+* Define the locations of the resources
+* Identify the namespace used by the required user code in the data structure
+
+You can also configure a `default` namespace, which can be used to provide resources when no specific {ucn} have been configured. For example, a data structure without an associated namespace, a partition loss listener, or when you are using an API that is not namespace-aware. For further information on using the `default` namespace, see the xref:clusters:ucn-non-associated.adoc[] topic.
+
+If you want to instantiate and register a customization that looks up code in a namespace, or submit a Jet job that uses previously-deployed code, you must also configure the permissions. For further information on permissions, see the xref:clusters:ucn-security.adoc[] and xref:security:native-client-security.adoc[] topics.
```

Comment:
Isn't [Jet + namespaces out of scope](https://hazelcast.atlassian.net/browse/HZ-3239)?